mount/mode/mode.go

```diff
@@ -0,0 +1,80 @@
+package mode
+
+import (
+	"strings"
+)
+
+type Mode struct {
+	modeString string `json:"modeString,omitempty"`
```

Comment:
Thank You,  Thank You , Thank You.  I have wasted more time on this.  Remind me to buy you a cookie next time I see you.  :^)